OloEditor/SandboxProject/Assets/Scripts/LuaScripts/LuaPlayerController.lua

```
@@ -0,0 +1,195 @@
+            end
+        end
+    end
+end
+
+function PlayerController.OnDestroy(id)
+    Log.Info("[LuaPlayerController] OnDestroy — entity " .. tostring(id))
+end
+
+-- Helpers (module-local)
+
+function FindNearestEnemy(myEntityID, maxRange)
```

Luacheck (1.2.0)

[warning] setting non-standard global variable 'FindNearestEnemy'

(W111)